backend/src/main/java/de/bund/digitalservice/ris/norms/config/SchedulerLockConfig.java

```diff
@@ -0,0 +1,38 @@
+package de.bund.digitalservice.ris.norms.config;
+
+import javax.sql.DataSource;
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * Configuration class for defining scheduler lock settings in the application. This class is
+ * annotated with {@link Configuration}, {@link EnableSchedulerLock} and {@link EnableScheduling} to
+ * activate the scheduler lock
+ */
+@Configuration
+@EnableScheduling
```

Comment:
We do have already this annotation in the app class, applied globably. Shoudn't we also add the @EnableSchedulerLock there and not in the config class?

Reply:
Guter Punkt! Ich tendiere dazu die Config in SchedulerConfig umzubenennen und alle Konfiguration dort zu machen. Damit trennen wir unsere Concerns und es ist analog zu SecurityConfig und SessionConfig.

Reply:
I see... and I see your point. Should we then also move the EnableScheduling to the config?

Reply:
Yes I will do that.